hello-world-microservice/pom.xml

```diff
@@ -10,7 +10,7 @@
     <version>${revision}${changelist}</version>
 
     <properties>
-        <revision>1015.0.368</revision>
+        <revision>1015.0.382</revision>
```

Comment:
I guess you mistakenly merged a wrong branch to your 1015 branch since it has release related commits. 
Can you get rid of the version diffs?

Reply:
Sorry, changed back.